src/modules/fundingManager/oracle/FM_PC_Oracle_Redeeming_v1.sol

```diff
@@ -717,11 +779,21 @@ contract FM_PC_Oracle_Redeeming_v1 is
         )
     {
         _ensureNonZeroTradeParameters(_depositAmount, _minAmountOut);
-        // Get protocol fee percentages and treasury addresses.
+
+        // Get protocol collateral sell fee percentage associated with the
+        // payment processor's processPayments function.
         (
-            address protocolCollateralTreasury,
-            address protocolIssuanceTreasury,
             uint protocolCollateralSellFeePercentage,
+            /* collateralTreasury */
+        ) = _getCollateralSellFeePercentage();
+
+        // Get protocol issuance sell fee percentage
+        (
+            /* protocolCollateralTreasury */
+            ,
+            address protocolIssuanceTreasury,
+            /* protocolCollateralSellFeePercentage */
+            ,
```

Comment:
Same comment as above: `protocolCollateralSellFeePercentage` appears in both functions which is confusing.

Reply:
Addressed